hcl2template/types.variables.go

```diff
@@ -669,8 +669,11 @@ func (cfg *PackerConfig) collectInputVariableValues(env []string, files []*hcl.F
 					Detail: fmt.Sprintf("A %q variable was set but was "+
 						"not found in known variables. To declare "+
 						"variable %q, place this block in one of your "+
-						".pkr files, such as variables.pkr.hcl",
-						name, name),
+						".pkr files, such as variables.pkr.hcl\n\n"+
```

Comment:
Lets call the file extension what it is to avoid any potential confusion on `.pkr` files. I can easily see someone thinking this to mean `.pkrvars` or `.pkr.hcl`
```suggestion
						".pkr.hcl files, such as variables.pkr.hcl\n\n"+
```